### DISCUSSION TITLE
svgtranscoder can not convert svg file to ImageVector class hao to do it

### DISCUSSION THREAD

**Discussion by ning-drift** [kirill-grouchnikov/aurora#73] [Q&A] [answered]
as title mean's. my project use ImageVector to display icon. how to do it. like some plugin?

**Comment by kirill-grouchnikov** [comment, marked as answer]
ImageVector is not a target for Aurora SVG transcoder.

You might want to explore https://github.com/DevSrSouza/svg-to-compose and https://plugins.jetbrains.com/plugin/18619-svg-to-compose
